features/kernel/src/main/feature/feature.xml

```diff
@@ -23,8 +23,33 @@
                  The karaf pax.web features (version 7.0.0) are blacklisted-->
     <repository>mvn:org.ops4j.pax.web/pax-web-features/${pax.web.version}/xml/features</repository>
 
+    <!-- overwrite karaf's spring with an upgraded version to address vulnerabilities -->
+    <feature name="spring" description="Spring 5.0.x support" version="${spring.version}_1">
+        <bundle dependency="true" start-level="30">
+            mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aopalliance/1.0_6
+        </bundle>
+        <bundle start-level="30">
+            mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-core/${spring.version}_1
```

Comment:
❓ Isn't our `spring.version` 5.0.4_RELEASE coming from `ddf-parent`? https://github.com/codice/ddf-parent/blob/DDF-Parent-1.0.0/pom.xml#L265

Reply:
Right, I need to update that first. thanks

Reply:
The spring version has been changed to 5.0.5.RELEASE in ddf-parent 1.0.2.